frontend/src/common/enums/app/csv.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum CSV {
```

Comment:
```suggestion
enum EAMCreateWorkerCSVColumn {
```
move to `frontend/src/common/enums/eam`